VoiceControl/VoiceControl.cpp

```diff
@@ -6,7 +6,7 @@
 #include "UtilsIarm.h"
 
 #define API_VERSION_NUMBER_MAJOR 1
-#define API_VERSION_NUMBER_MINOR 1
+#define API_VERSION_NUMBER_MINOR 2
```

Comment:
Pls also add CHANGELOG entry with user friendly information on the change. Pls don't mention JIRA tickets.

Reply:
Done.